src/main/java/uk/gov/justice/digital/delius/jpa/standard/entity/Contact.java

```diff
@@ -211,12 +221,20 @@ public boolean isRarActivity() {
 
         // the NSI is the primary component of RAR, so we check that first.
         // we also don't have to be careful with defaults here as the requirement and nsi foreign keys are mutually exclusive.
-        return Optional.ofNullable(getNsi())
+        return ofNullable(getNsi())
             .filter(nsi -> !nsi.getSoftDeleted() && nsi.isRarNsi())
             .map(Either::<Nsi, Requirement>left)
-            .or(() -> Optional.ofNullable(getRequirement())
+            .or(() -> ofNullable(getRequirement())
                 .filter(r -> !r.getSoftDeleted() && r.isRarRequirement())
                 .map(Either::<Nsi, Requirement>right))
             .orElse(null);
     }
+
+    public Optional<Enforcement> getEnforcement() {
+        if (enforcements.size() > 1) {
```

Comment:
the data model allows multiple, are you 100% sure in live we never have more than 1. If there is won't this mean the API will fail?

Reply:
Although data model allows this, in reality we would never have more than one enforcements for a contact, hence this validation. Confirmed by @axle-h

Reply:
OK, given that then just map it as a one-to-one

`
    @OneToOne(mappedBy = "contact")
     private Enforcement enforcement;
`

Reply:
Done